### ISSUE TITLE
Add Debian 10 Buster image

### ISSUE THREAD

**Issue by selfthinker** [nikolas-n/GNU-Linux-on-Asus-C201-Chromebook#8]
This [Debian 10 Buster image](https://zutshigroup.com/doku.php/tech:c100p:main) worked well on my Asus C100 (at least when booting from USB/SD). I suspect if it works for the C100 it will also work for the C201.

**Comment by kapouer** [comment]
Cool but really [this is more convenient](https://github.com/austin987/crawfishos)

**Comment by nikolas-n** [comment]
I agree with @kapouer actually. This is the distro I'm using at the moment. The caveat though is that it's not **free** (as in freedom). Apart from that, it works really well.

I'll add both distros anyway, when I find some time. Thanks for the info @selfthinker!

**Comment by selfthinker** [comment]
I thought the point of this repo is to collect all possible distros that work with the C201.
Why would you disagree to add something to that just because you prefer something else?

**Comment by selfthinker** [comment]
Ah, I responded when @nikolas-n's comment wasn't edited yet and was missing the second paragraph.
Thanks for planning to add both.

**Comment by kapouer** [comment]
> I thought the point of this repo is to collect all possible distros that work with the C201.
> Why would you disagree to add something to that just because you prefer something else?

Oh right ! Sorry if i was implying not to add something. That wasn't my intention.

**Comment by selfthinker** [comment]
@kapouer mentioning CrawfishOS reminds me that another PrawnOS clone is missing from the list: [Cadmium](https://github.com/Maccraft123/Cadmium).

**Comment by julio-guillen** [comment]
Hi. I wasnt able to install Cadmium but CrawfishOS worked flawlessly in my Librebooted C201. You can even create the sdcard/usb bootable devide from ChromeOS. Just use _shell_ and _sudo -s_ and then (for installing in sdcard) _dd if=CrawfishOS.img of=/dev/mmcblk1 status=progress; sync_ and you are done. Later, when booting from sdcard/usb, use root user and execute _InstallCrawfishOS_. thats all. Really easy to set up.

**Comment by kapouer** [comment]
Hi, I managed to boot cadmium from usb, install it, then rebuild linux-image 5.19.4 with debian's armhf kernel config + needed c201 patches. Now I have a standard debian/bookworm laptop running gnome smoothly with all the loadable kernel modules on root partition, as expected.
Special mention to https://github.com/alpernebbi/depthcharge-tools which, with very few configuration, installs my custom debian kernel packages smoothly on a/b partitions.
There are still some bugs left (sporadic black flashes on GL contexts, bluetooth crashes kernel, wake from suspend sometimes doesn't wake at all, but maybe that's only on libreboot).
